.github/ISSUE_TEMPLATE/new_issue_template.yml

```diff
@@ -0,0 +1,63 @@
+name: SS3 Team Issues - New Template
+description: Issue template for SS3 team members ONLY
+title: "[Issue]: "
```

Comment:
Can we put something different in the title than "[Issue]"? We already know it is an "issue" so if we want default text here for the SS3 team members to add on to, then I think it should be something that signifies that it is an issue that was filed under the team members only template but we can also do this with a tag if we don't want it in the issue name. Note that when I go to the create an issue using the template my cursor is automatically placed where the red arrow is; so, it is slightly confusing if the text in the square brackets should be replaced, added to where the cursor is, or move the cursor and added on to?
![image](https://user-images.githubusercontent.com/4108564/233117102-28008a84-9139-418d-b158-a0a2c94c018d.png)

Reply:
New commit has [SS3 Team Issue]: Title of Issue Here to let people know that they should create a title for the issue.